python/neuroglancer/pipeline/task_queue.py

```diff
@@ -98,10 +161,20 @@ def insert(self, task):
             "tag": task.__class__.__name__
         }
 
-        self.api.insert(project=self._project,
-                        taskqueue=self._queue_name,
-                        body=body).execute(num_retries=6)
+        def cloud_insertion(api):
```

Comment:
Maybe you already understood this. If you did, my opinion is that the convention is wrong.